cfme/tests/containers/test_relationships_tables.py

```diff
@@ -49,19 +50,23 @@ def test_pods_rel(provider, rel):
         lambda ch: 'nginx' not in ch and not ch.startswith('metrics'),
         ui_pods)
 
-    for name in ui_pods_revised:
+    for name in random.shuffle(ui_pods_revised):
         obj = Pod(name, provider)
-
         val = obj.get_detail('Relationships', rel)
-        if val == '0':
-            continue
-        obj.click_element('Relationships', rel)
+        if val != '0':
+            break
+        else:
+            # the row can't be clicked when there are no items, and has class 'no-hover'
+            logger.info('No items for node relationship: {}'.format(rel))
```

Comment:
Umn, maybe we can add TODO. Point (topic) of this patch is to deal with the tests running time.